ee/tabby-ui/lib/tabby-gql-client.ts

```diff
@@ -0,0 +1,15 @@
+import { GraphQLClient, Variables, RequestOptions } from 'graphql-request'
+
+export const graphQLClient = new GraphQLClient(
+  `${process.env.NEXT_PUBLIC_TABBY_SERVER_URL ?? ''}/graphql`,
+  {
+    credentials: 'include',
```

Comment:
cors work after removing this line - please research and see what's the reason behind.